addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/automation/OpenApiJob.java

```diff
@@ -83,8 +83,20 @@ public void verifyParameters(AutomationProgress progress) {
     }
 
     @Override
-    public void applyParameters(AutomationProgress progress) {
-        // Nothing to do
+    public boolean applyCustomParameter(String name, String value) {
```

Comment:
honestly I'm not sure if this is the right place.. This method is called right before running the job and the name does suggest something that might be right, but most implementations of `AutomationJob` don't implement this method.

If this is not the way to go, then what is this method supposed to do in general? What is its purpose?